tracker/tracker/src/main/app/observer/observer.ts

```diff
@@ -76,7 +69,10 @@ export default abstract class Observer {
           }
           if (type === 'childList') {
             for (let i = 0; i < mutation.removedNodes.length; i++) {
-              this.bindTree(mutation.removedNodes[i], true)
+              // Should be the same as bindTree(mutation.removedNodes[i]), but logic needs to be be untied
+              if (isObservable(mutation.removedNodes[i])) {
```

Comment:
wouldn't this add performance overhead?

Reply:
We can just discard all removed nodes I think

Reply:
It wouldn't.

Like without `isNew` ? 
later, it should be heavily tested. There is something that we don't see that resulted in bug in previous implementation.

Reply:
which one?

Reply:
Elements disappear.